rebasehelper/application.py

```diff
@@ -217,12 +217,16 @@ def _initialize_data(self):
         self.old_rest_sources = [os.path.abspath(x) for x in self.spec_file.get_sources()[1:]]
         self.new_rest_sources = [os.path.abspath(x) for x in self.rebase_spec_file.get_sources()[1:]]
 
-    def get_rpm_packages(self, dirname):
-        """
-        Function returns RPM packages stored in dirname/old and dirname/new directories
+    def get_rpm_packages(self, dirname: str) -> bool:
+        """Checks if there are RPM packages in dirname/old-build and
+        dirname/new-build. Saves the found packages to result store
```

Comment:
Hm, how about `Updates results store with build data found in the specified directory, if any.`?